src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -117,7 +117,9 @@ static void ApplyExtendedOpNodes(MultiExtendedOp *originalNode,
 								 MultiExtendedOp *masterNode,
 								 MultiExtendedOp *workerNode);
 static void TransformSubqueryNode(MultiTable *subqueryNode, List *tableNodeList);
-static MultiExtendedOp * MasterExtendedOpNode(MultiExtendedOp *originalOpNode);
+static MultiExtendedOp * MasterExtendedOpNode(MultiExtendedOp *originalOpNode, bool
```

Comment:
visual: we can take bool to the second line

Reply:
citus_indent, fixed